.editorconfig

```diff
@@ -0,0 +1,24 @@
+root = true
+
+[*]
+charset = utf-8
+# end_of_line = lf
+indent_style = space
+insert_final_newline = true
+max_line_length = 120
+
+[*.java]
+ij_continuation_indent_size = 4
+ij_java_class_count_to_use_import_on_demand = 999
+ij_java_doc_do_not_wrap_if_one_line = true
+ij_java_doc_preserve_line_breaks = true
+ij_java_imports_layout = $*, |, java.**, |, javax.**, |, org.**, |, com.**, |, *
+ij_java_line_comment_add_space = true
+ij_java_line_comment_at_first_column = false
+ij_java_names_count_to_use_import_on_demand = 999
+ij_java_packages_to_use_import_on_demand = unset
```

Comment:
```xml
    <option name="PACKAGES_TO_USE_IMPORT_ON_DEMAND">
      <value />
    </option>
```
its the same keys, @Anmol202005,  if not to much special stuff done here, its a easy job to adapt:
<img width="673" alt="image" src="https://github.com/user-attachments/assets/8c0e14de-e491-44bf-887c-16f6807298db" />